Ci compilation error: /__w/Kratos/Kratos/kratos/utilities/variable_utils.h:755:64: error: dependent-name 'TVariableArgs::Type' is parsed as a non-type, but instantiation yields a type

--- a/kratos/utilities/variable_utils.h
+++ b/kratos/utilities/variable_utils.h
@@ -752,7 +752,7 @@ public:
         const TVariableArgs&... rVariableArgs)
     {
         block_for_each(rNodes, [&](NodeType& rNode){(
-            AuxiliaryHistoricalValueSetter<TVariableArgs::Type>(rVariableArgs, rVariableArgs.Zero(), rNode), ...);
+            AuxiliaryHistoricalValueSetter<typename TVariableArgs::Type>(rVariableArgs, rVariableArgs.Zero(), rNode), ...);
         });
     }
 

